### DISCUSSION TITLE
Disable login?

### DISCUSSION THREAD

**Discussion by dbdmdbdmdbdm** [mealie-recipes/mealie#2545] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Would like to disable login globally for any users accessing the web UI. Given that this is only used locally on network, this would make it easier for household members to use without having to remember or provision another set of credentials. 

### Please provide a concise description of the feature that would resolve your issue.

env variable that allows disabling login

### Please consider and list out some caveats or tradeoffs made in your design decision

N/A

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
While not exactly the same, there's a PR in to make [browsing recipes public](https://github.com/mealie-recipes/mealie/pull/2525)
